app/viewmodels/pagination/PaginationViewModel.scala

```diff
@@ -0,0 +1,130 @@
+/*
```

Comment:
I can see we have some tests for the ListPaginationViewModel, do we need some for this as well?

Reply:
As I earlier mentioned, tests for this are covered as part of ListPaginationViewModel. ListPaginationViewModel uses apply method of PaginationViewModel and it does do any further processing apart from calling apply method of PaginationViewModel. The way trait PaginationViewModel is plugged and the definition of apply method of PaginationViewModel, it can be tested by this way. Let me know if you have further queries